packages/react-core/src/components/Select/SelectMenu.tsx

```diff
@@ -77,7 +77,7 @@ class SelectMenuWithRef extends React.Component<SelectMenuProps> {
     const { selected, sendRef, keyHandler } = this.props;
     const isSelected =
       selected && selected.constructor === Array
-        ? selected && (Array.isArray(selected) && selected.includes(child.props.value))
+        ? selected && Array.isArray(selected) && selected.includes(child.props.value)
```

Comment:
@redallen I didn't intentionally change this - the linter did - lemme know if you want me to revert

Reply:
It's fine!